jujugui/static/gui/src/app/app.js

```diff
@@ -678,6 +671,7 @@ YUI.add('juju-gui', function(Y) {
         if (this.env.get('modelUUID')) {
           return;
         }
+        const current = state.current;
```

Comment:
I'm assuming that this has been moved here because of the possible `changeState` calls above? Could you please add a comment to that effect if that's so as it's not obvious.